<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="John" failures="1" tests="1" name="Admin Test" time="0.008" errors="0" timestamp="15 Feb 2023 15:07:58 GMT">
  <testcase classname="com.imb.tests.AdminTest" name="m1" time="0.002">
    <failure type="java.lang.ArithmeticException">
      <![CDATA[java.lang.ArithmeticException
at com.imb.tests.AdminTest.m1(AdminTest.java:8)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- m1 -->
</testsuite> <!-- Admin Test -->
